### PR TITLE
docs: update oudated docs

### PR DESCRIPTION
**Pull request by tujoworker**
Update outdated docs

### CODE REVIEW

**Review by langz — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/design-system/about.md [hunk 1]
```diff
@@ -37,18 +37,21 @@ You can read more about what we at DNB UX expect and define as the minimal [acce
...
 
-- [Jens Thuland](https://dnb.enterprise.slack.com/user/@WE4QCR6PQ)
-- [Tobias Høegh](https://dnb.enterprise.slack.com/user/@WE2M4E65N)
-- [Casper Brekke](https://dnb.enterprise.slack.com/user/@WDU4VCEP5)
+- Jens Thuland
+- Casper Brekke
+- Tobias Høegh
 - Sindre Marken
+- Dina Rosvoll
+- Anders Langseth
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/design-system/about.md:45]: 🎊

Comment (thaytharma) [reply packages/dnb-design-system-portal/src/docs/design-system/about.md:45]: 😍

**Review by thaytharma — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/design-system/about.md (hunk 1 above)
Comment (dinarosv) [reply packages/dnb-design-system-portal/src/docs/design-system/about.md:45]: 🥳  🙏  ❤️

**Review by dinarosv — CHANGES_REQUESTED** [review]
Thanks for updating this!

There's a section for this in Contribution as well, I will add that to this PR asap

**Review by dinarosv — APPROVED** [review]
Removed duplicate contributors page under Contribution guide and linked to the one you updated instead. Hope that was okay!

Please check over before merge 😊

**Comment by tujoworker** [comment]
> Removed duplicate contributors page under Contribution guide and linked to the one you updated instead. Hope that was okay!
> 
> Please check over before merge 😊

Nice – very good ✨

**Pull request merged**